src/BingChat/BingChatClient.cs

```diff
@@ -107,11 +115,15 @@ public async Task<BingChatConversation> CreateConversation()
             throw new BingChatException(message);
         }
 
+        var encryptedConversationSignature = rawResponse.Headers.GetValues("X-Sydney-EncryptedConversationSignature").FirstOrDefault();
```

Comment:
If this doesn't work with `encryptedConversationSignature`, shouldn't it be `First()` instead of `FirstOrDefault()`?